doc/advanced_usage.md

```diff
@@ -92,7 +92,52 @@ with the data that was posted.
 If the posting was unsuccessful,
 the status would be 400.
 
+## Upgrading an existing Taxonomy
 
+Taxonomies are used for classification. There is typically a sitewide taxonomy that all users of SkyPortal can see. By default, we seed a new SP database with the latest taxonomy from the [Time-domain Astronomy Taxonomy
+ (Github)](https://github.com/profjsb/timedomain-taxonomy) (`tdtax`). From time to time, the latest taxonomy may be upgraded as new subclasses of sources are discovered and SP admins may need to push a new version of the taxonomy to the live application.
+
+To upgrade to the latest `tdtax`, as an admin, you will need to generate a token with a "Post taxonomy" ACL. Next install the latest `tdtax` on your system:
+
+```
+pip install -U tdtax
+```
+
+Next, in Python:
+
+```python
+import requests
+import tdtax
+
+tax_obj =  {'name': 'Sitewide Taxonomy',
+            'provenance': 'https://github.com/profjsb/timedomain-taxonomy',
+            'hierarchy': tdtax.taxonomy,
+            'version': str(tdtax.__version__),
+}
+
+token = "USE-YOUR-TOKEN-HERE"
+def api(method, endpoint, data=None):
+    headers = {'Authorization': f'token {token}'}
+    response = requests.request(method, endpoint, json=data, headers=headers)
+    return response
+
+response = api('POST',
+               'URL_TO_SKYPORTAL_INSTANCE/api/taxonomy',
+                data=tax_obj)
+
+print(response.json())
+```
+You should then see something like:
+
+```
+{'status': 'success',
+ 'data': {'taxonomy_id': 9},
+ 'version': '0.9.dev0+git...'}
+```
+
+If the `name` of the taxonomy is the exactly the same as a taxonomy already in the system, this new taxonomy will supercede that taxonomy. That is, by default it will be set to be the latest version (`IsLatest = True`) and shown to end users in the dropdown menu on the Source page. Older classifications will be still associated with the `taxonomy_id` they were originally connected to.
```

Comment:
```suggestion
If the `name` of the taxonomy is exactly the same as a taxonomy already in the system, this new taxonomy will supercede that taxonomy. That is, by default it will be set to be the latest version (`IsLatest = True`) and shown to end users in the dropdown menu on the Source page. Older classifications will be still associated with the `taxonomy_id` they were originally connected to.
```